staging/src/k8s.io/api/testdata/HEAD/core.v1.Node.json

```diff
@@ -42,113 +42,115 @@
   },
   "spec": {
     "podCIDR": "24",
-    "providerID": "25",
-    "unschedulable": true,
+    "podCIDRs": [
```

Comment:
@liggitt is it expected that this just regenerates the whole file?

Reply:
for HEAD, when fields are added, yes